cogstat/cogstat_hyp_test.py

```diff
@@ -498,6 +498,73 @@ def variable_pair_hyp_test(data, x, y, meas_lev, normality=None, homoscedasticit
         population_result += chi_squared_test(data, x, y)
     return population_result
 
+def multiple_regression_hyp_tests(result, x, data, normality, homoscedasticity, multicollinearity):
+    """Hypothesis tests for model and regressor slopes in multiple linear regression.
+
+    Parameters
+    ----------
+    result : statsmodels regression result object
+        The result of the multiple regression analysis.
+    x : list of str
```

Comment:
please, use another variable name

does the function work with one regressor? If so, please, extend the docstring. If not, please extend the code and the docstring :)